src/chrome/content/rules/kf5.com.xml

```diff
@@ -8,17 +8,25 @@
 <ruleset name="kf5.com">
 
 	<target host="kf5.com" />
-	<target host="assets.kf5.com" />
-	<target host="assetscdn.kf5.com" />
-	<target host="blog.kf5.com" />
-	<target host="developer.kf5.com" />
-	<target host="qiniu.kf5.com" />
-	<target host="support.kf5.com" />
-	<target host="www.kf5.com" />
+	
+	<target host="*.kf5.com" />
+
+	<exclusion pattern="^http://(agents|d)\.kf5\.com/(.*)" />
```

Comment:
The `(.*)` on the end is not needed, the pattern will match anyway and is much simpler.